packages/cli/package.json

```diff
@@ -29,12 +29,12 @@
   },
   "dependencies": {
     "@float-capital/float-subgraph-uncrashable": "^0.0.0-alpha.4",
+    "@graphprotocol/graph-tooling-napi-utils": "workspace:*",
```

Comment:
the issue we are facing now is even for test suite we have to build otherwise it will fail 🤔

Reply:
yeah, but we build it in the CI, maybe locally we should pre-build before testing too